src/store.js

```diff
@@ -3,52 +3,72 @@ import Vue from 'vue'
 Vue.use(Vuex)
 
 const state = {
-  sex:"",
+  gender:"",
   birthday:"",
   question:"",
+  question2nd:"",
+  question3rd:"",
   consultation:""
 }
 
 const mutations = {
-  setSex(state,val){
-    state.sex = val
+  setGender(state,val){
+    state.gender = val
   },
   setBirthday(state,val){
     state.birthday = val
   },
   setQuestion(state,val){
     state.question = val
   },
+  setQuestion2nd(state,val){
+    state.question2nd = val
+  },
+  setQuestion3rd(state,val){
+    state.question3rd = val
+  },
   setConsultation(state,val){
     state.consultation = val
   },
 }
 
 const actions = {
-  updateSex(commit,e){
-    commit('setSex',e.value)
+  setGender(context,val){
+    context.commit('setGender',val)
+  },
+  setBirthday(context,val){
+    context.commit('setBirthday',val)
   },
-  updateBirthday(commit,e){
-    commit('setBirthday',e.value)
+  setQuestion(context,val){
+    context.commit('setQuestion',val)
   },
-  updateQuestion(commit,e){
-    commit('setQuestion',e.value)
+  setQuestion2nd(context,val){
+    context.commit('setQuestion2nd',val)
```

Comment:
[actionの記事](https://vuex.vuejs.org/ja/guide/actions.html)にも記載がありますが、第一引数は以下のようにするとcontextの記述を省略できますのでおすすめです。
```suggestion
  setQuestion2nd({commit},val){
    commit('setQuestion2nd',val)
```

ちなみに`{commit}`は分割代入という方法です。
https://developer.mozilla.org/ja/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment